src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -150,6 +161,39 @@ public void setTransaction(Transaction target, Transaction editedTransaction) {
      */
     public void removeTransaction(Transaction key) {
         transactions.remove(key);
+        sortPersons();
+    }
+
+    /**
+     * Sets sort person to descending.
+     */
+    public void setDesc() {
```

Comment:
Can consider rename these methods to be more meaningful, considering it is sorting *people* by their *balance* cause there might be other sorting methods in the future